pkg/ebpf/bytecode/tracer-ebpf.go

```diff
@@ -867,7 +867,7 @@ func bindataPkgEbpfCTcpqueuelengthkernuserh() (*asset, error) {
 		name:        "pkg/ebpf/c/tcp-queue-length-kern-user.h",
 		size:        430,
 		md5checksum: "",
-		mode:        os.FileMode(420),
+		mode:        os.FileMode(436),
```

Comment:
I don't think this needs to change.

Reply:
This keeps getting put in every time I build. I've seen other PR's with this change as well. Do you know what this changed?

Reply:
Are you building within the VM? it is likely a `umask` difference.

Reply:
Yes, building inside the VM.

Reply:
I would do a quick `chmod 420 pkg/ebpf/c/tcp-queue-length-kern-user.h` then.

Reply:
That didn't work. I am just going to revert this change manually.